### PR TITLE
Add filter branch v1.1

### PR DESCRIPTION
**Pull request by NearTruth**

### CODE REVIEW

File: cron_jobs/report_space_cron.sh [hunk 1]
```diff
@@ -12,19 +12,34 @@ NCDU_JSONS_DIRECTORY=ncdu_jsons
 NCDU_EXPORT_REPO=git_repos/ncdu-export
 KARSANLAB_GIT_REPO=git_repos/KARSANBIO-3374_space-monitoring-with-email
 TRACKING_TABLE=on_going_space_report.tsv
+
```
Comment (joshbridgers) [inline cron_jobs/report_space_cron.sh:15]: Keep all magic strings in one block and sort alphabetically

File: cron_jobs/report_space_cron.sh [hunk 2]
```diff
@@ -12,19 +12,34 @@ NCDU_JSONS_DIRECTORY=ncdu_jsons
...
+      --exclude /projects/karsanlab/dtam/KARSANBIO-3003_Attempt_variant_calling_on_mtscATAC-seq_libraries/lib/varCA/ \
+      --exclude /projects/karsanlab/dtam/anaconda3-linux/ \
+      --exclude /projects/karsanlab/sgillis/CHARM/CHARM-67_Running_CHARM_pipelines/CHARM-82_Run_UHN_cfMeDIP-Seq_QC/ \
+      --exclude /projects/karsanlab/sgillis/CHARM/CHARM-80_cfMeDIP_cell-line_validation_sequencing_analysis/cell-line_validation/ \
+      --exclude /projects/karsanlab/sgillis/software/ \
+      --exclude /projects/karsanlab/shuang/softwares/ \
+      --exclude /projects/karsanlab/software/ \
+      --exclude /projects/karsanlab/tliu/Dry-lab/KARSANBIO-3051_AML-alternative-splicing/data/ \
+      --exclude /projects/karsanlab/tliu/Dry-lab/KARSANBIO-3351_Aberrant-Splicing-Events-of-AML/data/ \
+    -o - ${DIRECTORY_ROOT}  | gzip > ${NCDU_JSONS_DIRECTORY}/${DATE}_ncdu.json.gz
```
Comment (joshbridgers) [inline cron_jobs/report_space_cron.sh:42]: move "-o - ${DIRECTORY_ROOT}" to before the exclude list.

File: cron_jobs/report_space_cron.sh [hunk 3]
```diff
@@ -12,19 +12,34 @@ NCDU_JSONS_DIRECTORY=ncdu_jsons
...
-echo Started space audit at $(date -Iseconds)
+echo Started space audit at "$(date -Iseconds)"
 
 cd ${SPACE_MONITORING_DIRECTORY}
 eval "$(/projects/karsanlab/software/linux-x86_64-centos7/anaconda3/bin/conda shell.bash hook)"
 conda activate ./env_space_monitoring
 mkdir -p ${PARSED_DATA_DIRECTORY}/${DATE}
-ncdu -0 -x -e -r --exclude ${EXCLUDE_DIRECTORY} -o - ${DIRECTORY_ROOT}  | gzip > ${NCDU_JSONS_DIRECTORY}/${DATE}_ncdu.json.gz
+ncdu -0 -x -e -r --exclude ${EXCLUDE_DIRECTORY} \
+      --exclude /projects/karsanlab/CCG_panel_resources/CCG_Pipeline_LySeq_aSHM/lib/python3.6/ \
```
Comment (joshbridgers) [inline cron_jobs/report_space_cron.sh:30]: Will this command error our if the directory does not exist anymore?

File: scripts/send_emails.py [hunk 4]
```diff
@@ -15,7 +15,8 @@
 PARSED_DATA_DIRECTORY = f"/projects/karsanlab/space_monitoring/parsed_data"
 # sets the directory where the summary tsv files are stored
 # TODO change this so that the directory can be configured from outside
-
+# link to the wiki documentation page
```
Comment (joshbridgers) [inline scripts/send_emails.py:18]: comment is redundant with variable name

File: scripts/test/test_parse_ncdu_output_email.py [hunk 5]
```diff
@@ -8,29 +8,34 @@
...
-        tests that parse_json_from_file correctly captures the contents of test_ncdu_small.flat.json file and sorts the information of the problematic files into their respective lists. 
+        tests that parse_json_from_file correctly captures the contents of test_ncdu_small.flat.json file and sorts the
+        information of the problematic files into their respective lists.
+
+        check that files and directories included in the ignore_list are excluded from the final lists
 
         checks for the content and order of each list
         """
-        input_file = "../test_json/test_ncdu_small.flat.json"
+        ignore_list = ["/projects/karsanlab/exclude_dir", "/projects/karsanlab/exclude_file1"]
```
Comment (joshbridgers) [inline scripts/test/test_parse_ncdu_output_email.py:23]: These directories don't exist. Is this used?

**Review by joshbridgers — CHANGES_REQUESTED** [review]

File: scripts/test/test_parse_ncdu_output_email.py (hunk 5 above)
Comment (NearTruth) [reply scripts/test/test_parse_ncdu_output_email.py:23]: That list is unused, I will delete it.

File: cron_jobs/report_space_cron.sh (hunk 2 above)
Comment (NearTruth) [reply cron_jobs/report_space_cron.sh]: done.

File: cron_jobs/report_space_cron.sh (hunk 1 above)
Comment (NearTruth) [reply cron_jobs/report_space_cron.sh:19]: done.

File: scripts/send_emails.py (hunk 4 above)
Comment (joshbridgers) [reply scripts/send_emails.py:18]: Delete this comment

**Review by joshbridgers — APPROVED** [review]

**Pull request merged**